### DISCUSSION TITLE
如何在目录中添加”摘要“和”ABSTRACT“

### DISCUSSION THREAD

**Discussion by zw-yu** [sjtug/SJTUThesis#942] [提问] [answered]
main.tex中部分代码如下：

\maketitle
% 原创性声明及使用授权书
%\copyrightpage
% 插入外置原创性声明及使用授权书
% 此时必须在导言区使用 \usepackage{pdfpages}
\copyrightpage[scans/last.pdf]
% 前置部分
\frontmatter
% 摘要
\input{contents/abstract}
% 目录
\tableofcontents
\mainmatter

而abstract.tex中的代码如下：
\begin{abstract}[zh]
  随着互联网的发展。。。
\end{abstract}
\begin{abstract}[en]
  With the development of the internet, 
\end{abstract}

我想不改变摘要与目录的相对位置，而把中英文摘要加入目录中

**Comment by LogCreative** [comment, marked as answer]
```latex
\begin{abstract}[zh]
\addcontentsline{toc}{chapter}{摘要}

\end{abstract}

\begin{abstract}[en]
\addcontentsline{toc}{chapter}{ABSTRACT}

\end{abstract}
```
